# Day 28
In Day 28 we learn about Jumps in JavaScript.

### The throw statement
The ```throw``` statement throws a user-defined exception. Execution of the current function will stop (the statements after throw won't be executed), and control will be passed to the first catch block in the call stack. If no catch block exists among caller functions, the program will terminate.

**Syntax:**
```throw expression ```
